infoschema/tables.go

```diff
@@ -2226,3 +2234,142 @@ func (vt *VirtualTable) GetPhysicalID() int64 {
 func (vt *VirtualTable) Type() table.Type {
 	return table.VirtualTable
 }
+
+// GetTiFlashServerInfo returns all TiFlash server infos
+func GetTiFlashServerInfo(sctx sessionctx.Context) ([]ServerInfo, error) {
+	if config.GetGlobalConfig().DisaggregatedTiFlash {
```

Comment:
DisaggregatedTiFlash is not supported because pd does not have related information?

Reply:
It is still under design, especially for auto scaling.

Reply:
When AutoScaler manager all tiflash_compute nodes, PD cannot get the server info of tiflash_compute, we need add API in AutoScaler to support this.